bpf/cpu/cpu.bpf.c

```diff
@@ -862,6 +893,8 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
     }
     unwind_state->unwinding_jit = false;
 
+    // We ideally want RBP_TYPE_OFFSET here for DWARF unwinding, RBP_TYPE_REGISTER or RBP_TYPE_EXPRESSION
```

Comment:
I think this comment might be unnecessary as I think the code shows this already (and comments can go out of sync)